pkg/backup/backup_compaction_test.go

```diff
@@ -270,6 +271,48 @@ crdb_internal.json_to_pb(
 			t, db, []string{"foo"}, "'nodelocal://1/backup'", opts,
 		)
 	})
+
+	t.Run("pause resume and cancel", func(t *testing.T) {
+		db.Exec(t, "CREATE TABLE foo (a INT, b INT)")
+		defer func() {
+			db.Exec(t, "DROP TABLE foo, bar")
+		}()
+		db.Exec(t, "INSERT INTO foo VALUES (1, 1)")
+		start := getTime()
+		db.Exec(t, fmt.Sprintf(fullBackupAostCmd, start))
+		db.Exec(t, "INSERT INTO foo VALUES (2, 2)")
+		db.Exec(t, incBackupCmd)
+		db.Exec(t, "CREATE TABLE bar (a INT, b INT)")
+		db.Exec(t, "INSERT INTO bar VALUES (1, 1)")
```

Comment:
why does this test require 4 inc backups?

Reply:
Oh I suppose it doesn't, I've gotten into the habit of trying to make sure there are "meaningful" compactions, but I suppose the other tests cover that so there isn't really a need for it in this test.